tdwii_plus_examples/handlers.py

```diff
@@ -771,4 +772,45 @@ def handle_ncreate(event, storage_dir, db_path, cli_config, logger):
         finally:
             session.close()
 
+    # Database successfully updated, notify any globally subscriber
+    for globalsubscriber in _global_subscribers:
+        # Request association with subscriber
+        # get AET of UPS Event SCP (which is the AET if UPS Watch SCP)
+        acceptor =  event.assoc.acceptor
+        logger.info(f"UPS Event SCP AET: {acceptor.ae_title}")
+        
+        ae = AE(ae_title=acceptor.ae_title)
+        # hard code for the moment, deal with configuration of AE's soon
+        assoc = ae.associate(
+            "127.0.0.1",
+            11112,
+            contexts=UnifiedProcedurePresentationContexts,
+            ae_title=globalsubscriber,
+            max_pdu=16382,
+        )
+        
+        if assoc.is_established:
+            try:
+                logger.info(f"Send UPS State Report: {ds.SOPInstanceUID}, {ds.ProcedureStepState}")
+                event_info = Dataset()
+                event_info.ProcedureStepState = ds.ProcedureStepState
```

Comment:
CC.2.5.2 Service Class User Behavior:
The SCU shall specify a value of "SCHEDULED" for the Attribute Procedure Step State (0074,1000) in the N-CREATE request
primitive.
so... type 1, even though it isn't listed in Table CC.2.5-3

Reply:
solved in #32